EIPS/eip-5023.md

```diff
@@ -0,0 +1,144 @@
+---
+eip: 5023
+title: Shareable non-transferable non-fungible token
+description: An interface for shareable non-transferable NFTs
```

Comment:
This description is repeating the title a bit too closely. What is a shareable non-transferable NFT? Maybe something like:

```suggestion
description: Interface for value-holding tokens that are shared by multiple owners instead of transferred
```

Reply:
Updated in the PR.